RefRed/file_loading_utility.py

```diff
@@ -2,36 +2,16 @@
     TODO: verify whether this is really needed
 """
 
+# third party packages
+import numpy as np
 
-def loadCsvFile(filename):
-    '''
-    Loading CSV file created with Excel to test PeakFinderDerivation algorithm
-    '''
-    f = open(filename, 'rt')
-    data = f.readlines()
-    data  = data[0].split('\r')
-    final_data = []
-    for row in data:
-        _row_split = row.split(',')
-        final_data.append(_row_split)
-    f.close()
-    final_data = convertArrayToFloat(final_data)
-    [x, y, e] = isolateXYE(final_data)
-    return [x, y, e]
+# standard packages
+from typing import List
```

Comment:
Has RefRed been upgraded to python 3.5+ already?

Reply:
I think you're right and I'll have to drop the types for the present